src/huggingface_hub/inference/_generated/types/zero_shot_image_classification.py

```diff
@@ -0,0 +1,51 @@
+# Inference code generated from the JSON schema spec in @huggingface/tasks.
+#
+# Using ./src/scripts/inference-codegen
```

Comment:
(nit) tried to understand where this was coming from, maybe clarify with `huggingface.js/packages/tasks/scripts
/inference-codegen.ts` ?